domain/src/main/java/com/thoughtworks/go/domain/MaterialInstance.java

```diff
@@ -151,4 +151,20 @@ public String getUrl() {
     public String getUsername() {
         return username;
     }
+
+    public String getView() {
```

Comment:
Do we need these changes now?

Reply:
These are used for verification step in `MaterialRepositoryIntegrationTest`